test/com/jambit/DatabaseConnectionTest.java

```diff
@@ -81,4 +104,28 @@ void deleteData() throws SQLException {
     System.out.println(sql);
     st.executeUpdate(sql.toString());
   }
+
+  @AfterAll
+  static void close() throws SQLException {
+    if (DatabaseConnection.getDatabaseDriver() == DatabaseConnection.databaseDrivers.h2) {
+      databaseConnection
+          .getActiveDatabaseConnection()
+          .createStatement()
+          .executeUpdate("DROP TABLE MoodMeter;");
+    }
+  }
```

Comment:
You should not need that, since H2 is only in memory and after the test the whole database should be killed.